draft-ietf-quic-tls.md

```diff
@@ -532,9 +532,10 @@ older than 1.3 is negotiated.
 
 ## ClientHello Size
 
-QUIC requires that the initial handshake packet from a client fit within a
-single packet of at least 1280 octets.  With framing and packet overheads this
-value could be reduced.
+QUIC requires that the initial handshake packet from a client fit within the
+payload of a single packet.  A minimum-sized initial packet in IPv6 has space
+for a single TLS record of 1197 octets once the QUIC header and minimal framing
+is added.
```

Comment:
Ekr: There's 16 octets of FNV hash. I think it's off too though, since adding 16 bytes to your number (which I believe is correct otherwise) gives 1233 bytes.

I don't think it's necessary to articulate how many bytes of QUIC payload, since it just gets confusing. I think it's enough to say 1252 and 1232 bytes of UDP payload (or QUIC packet size) for v4 and v6. I think it's useful and important to separate v4 and v6 -- 20 bytes in the common v4 case is a fair bit.